services/minespace-web/src/components/dashboard/mine/noticeOfDeparture/NoticeOfDepartureTable.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Button, Table } from "antd";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+
+const columns = [
+  {
+    title: "Title",
+    dataIndex: "nod_title",
+    key: "nod_title",
+    sorter: (a, b) => (a.nod_title > b.nod_title ? -1 : 1),
+  },
+  {
+    title: "NOD #",
+    dataIndex: "nod_guid",
+    key: "nod_guid",
+    sorter: (a, b) => (a.nod_guid > b.nod_guid ? -1 : 1),
+  },
+  {
+    title: "Permit #",
+    dataIndex: ["permit", "permit_no"],
+    key: ["permit", "permit_no"],
+    sorter: (a, b) => (a.permit.permit_no > b.permit.permit_no ? -1 : 1),
+  },
+  {
+    title: "Type",
+    dataIndex: "type",
+    key: "type",
+    sorter: (a, b) => (a.type > b.type ? -1 : 1),
+  },
+  {
+    title: "Status",
+    dataIndex: "status",
+    key: "status",
+    sorter: (a, b) => (a.status > b.status ? -1 : 1),
+    defaultSortOrder: "ascend",
+  },
+  {
+    render: () => (
+      <Button
+        type="primary"
+        size="small"
+        onClick={() => {
+          // TODO: Create/Integrate Modal
```

Comment:
I notice you have created a modal, should it not be hooked up here?

Reply:
The modal that exists is for creating a new notice of departure.  What's needed here is a modal for viewing an existing one.

Reply:
Ah okay, thank you for the clarification.